pkg/cmd/resource/resource.go

```diff
@@ -0,0 +1,98 @@
+package resource
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	"github.com/stripe/stripe-cli/pkg/ansi"
+)
+
+//
+// Public types
+//
+
+// ResourceCmd represents resource commands. Resource commands can be either
+// top-level commands or nested under namespace commands. Resource commands
+// are containers for operation commands.
+type ResourceCmd struct { //nolint:golint
```

Comment:
OOC, why did this have to be nolinted?

Reply:
Because the package's name is `resource`, golint thinks `resource.ResourceCmd` is a redundant name and the struct should be named `resource.Cmd`.

Reply:
Ah, I see.